src/plugins/theme/screens.js

```diff
@@ -3,13 +3,11 @@
  */
 module.exports = function () {
   return {
-    xs: '0px',
```

Comment:
Why no xs option?

Reply:
In TW's default config, they don't explicitly include the 0 for the smallest breakpoint. I might change my mind later about some of these things though.